mcp/sensors/acs.c

```diff
@@ -372,8 +383,9 @@ extern unsigned int sched_lst; /* sched_lst */
 struct ACSDataStruct ACSData;
 
 /**
- * Reads the 5Hz data from the most recent frame received from UEIs and stores
- * it into the ACSData structure for use in pointing
+ * @brief Reads the 5Hz data from the most recent frame and stores
+ * it into the ACSData structure for use in pointing, as well as telemetry
+ * channels.
```

Comment:
add context for multiplication with 15000 in lines ~460

Reply:
changed to a #define but not sure what this is anyway. todo shubh